services/mailservers/api_test.go

```diff
@@ -65,9 +66,9 @@ func TestTopic(t *testing.T) {
 	defer close()
 	topicA := "0x61000000"
 	topicD := "0x64000000"
-	topic1 := MailserverTopic{Topic: topicA, LastRequest: 1}
-	topic2 := MailserverTopic{Topic: "0x6200000", LastRequest: 2}
-	topic3 := MailserverTopic{Topic: "0x6300000", LastRequest: 3}
+	topic1 := MailserverTopic{ContentTopic: topicA, LastRequest: 1}
+	topic2 := MailserverTopic{ContentTopic: "0x6200000", LastRequest: 2}
+	topic3 := MailserverTopic{ContentTopic: "0x6300000", LastRequest: 3}
```

Comment:
Is it possible to introduce testing that covers the new functionality introduced here? Example queries on the `pubsubTopic` and `contentTopic`.

Reply:
I will try!